packages/main/test/pages/TabContainer.html

```diff
@@ -726,6 +726,44 @@ <h2>Text only Start And End Overflow Custom Overflow Buttons</h2>
 				<ui5-button>Button 1</ui5-button>
 			</ui5-tab>
 		</ui5-tabcontainer>
+
+		<section>
+			<ui5-title>Select tabs programatically</ui5-title>
+			<ui5-tabcontainer id="tcSmall" style="width: 200px;">
+				<ui5-tab id="firstTab" text="First tab">
+					<ui5-button>Button 1</ui5-button>
+				</ui5-tab>
+				<ui5-tab text="Second tab">
+					<ui5-button>Button 1</ui5-button>
+				</ui5-tab>
+				<ui5-tab text="Third tab" selected>
+					<ui5-button>Button 1</ui5-button>
+				</ui5-tab>
+				<ui5-tab text="Fourth tab">
+					<ui5-button>Button 1</ui5-button>
+				</ui5-tab>
+				<ui5-tab text="Fifth tab">
+					<ui5-button>Button 1</ui5-button>
+				</ui5-tab>
+				<ui5-tab text="Sixth tab">
+					<ui5-button>Button 1</ui5-button>
+				</ui5-tab>
+				<ui5-tab id="nestedTabParent" text="Seventh tab">
+					<ui5-button>Button 1</ui5-button>
+
+					<ui5-tab id="nestedTab" slot="subTabs" text="Nested tab">
+						<ui5-button>Button 1</ui5-button>
+					</ui5-tab>
+				</ui5-tab>
+				<ui5-tab id="lastTab" text="Eighth tab">
+					<ui5-button>Button 1</ui5-button>
+				</ui5-tab>
+			</ui5-tabcontainer>
+		
+			<ui5-button id="selectFirst">Select first tab</ui5-button>
```

Comment:
I suggest title case for button labels:
			<ui5-button id="selectFirst">Select First Tab</ui5-button>
			<ui5-button id="selectLast">Select Last Tab</ui5-button>
			<ui5-button id="selectNested">Select a Nested Tab</ui5-button>